opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/TaskRegisterFragmentPresenter.java

```diff
@@ -412,11 +412,7 @@ public void onEventFound(Event event) {
                 JSONObject formJSON = getView().getJsonFormUtils().getFormJSON(getView().getContext(), formName, getTaskDetails(), getStructure());
                 getView().getJsonFormUtils().populateForm(event, formJSON);
                 getView().getJsonFormUtils().populateFormWithServerOptions(formName,formJSON);
-                if (IRS.equals(getTaskDetails().getTaskCode()) && NAMIBIA.equals(BuildConfig.BUILD_COUNTRY)) {
-                    formInteractor.findSprayDetails(IRS, getStructure().getId(), formJSON);
-                } else {
-                    getView().startForm(formJSON);
-                }
+                getView().startForm(formJSON);
```

Comment:
I think this does not solve the problem. You should check if `formInteractor.findSprayDetails` has bugs